src/Controller/Admin/AdminCandidaciesGroupCandidateImportController.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\LocalElection\CandidaciesGroup;
+use App\Entity\LocalElection\Candidacy;
+use App\Form\Admin\LocalElection\CandidateImportType;
+use App\LocalElection\CandidateImportCommand;
+use App\Repository\LocalElection\CandidacyRepository;
+use App\ValueObject\Genders;
+use Doctrine\ORM\EntityManagerInterface;
+use League\Csv\Exception;
+use League\Csv\Reader;
+use Sonata\AdminBundle\Controller\CRUDController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class AdminCandidaciesGroupCandidateImportController extends CRUDController
+{
+    private const CIVILITIES = [
+            Genders::MALE => ['homme', 'monsieur', 'm', 'mr', 'masculin'],
+            Genders::FEMALE => ['femme', 'madame', 'mme', 'féminin'],
+    ];
+
+    public function __construct(private readonly EntityManagerInterface $em)
+    {
+    }
+
+    public function candidateImportAction(
+        Request $request,
+        CandidaciesGroup $candidaciesGroup,
+        CandidacyRepository $candidacyRepository,
+        ): Response {
+        $this->admin->checkAccess('candidate_import');
+
+        $form = $this
+            ->createForm(CandidateImportType::class, $candidateImportCommand = new CandidateImportCommand())
+            ->handleRequest($request)
+        ;
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            if (!$file = $candidateImportCommand->file?->getPathname()) {
+                $this->addFlash('sonata_flash_error', 'Aucun fichier trouvé');
+
+                return $this->redirect($this->admin->generateObjectUrl('candidate_import', $candidaciesGroup));
+            }
```

Comment:
I think we can remove this validation, because your `@Assert\NotBlank` already takes care of that. 
```suggestion
```